src/getHumanAge.test.js

```diff
@@ -1,8 +1,61 @@
+'use strict';
+
 describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
   test('should be declared', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test(`should return 0 for 0 cat's and dog's years`, () => {
```

Comment:
```suggestion
  test(`should return [0, 0] for 0 cat's and dog's years`, () => {
```

it seems like the function returns a number, not an array